mal/useranimelist_test.go

```diff
@@ -150,6 +154,8 @@ func TestAnimeServiceUpdateMyListStatus(t *testing.T) {
 		Tags:               []string{"foo", "bar"},
 		Comments:           "comments",
 		UpdatedAt:          time.Date(2018, 04, 25, 15, 59, 52, 0, time.UTC),
+		StartDate:          "2022-02-20",
+		FinishDate:         "2022-04-16",
```

Comment:
We could leave this empty to show explicitly that we are expecting the value to be empty or we could even remove the line entirely.
```suggestion
		FinishDate:         "",
```